conf/config_template.yaml

```diff
@@ -13,6 +13,7 @@ global:
   samples_size: 512
   num_classes: 5
   data_path: path/to/data
+  mlflow_uri: path/to/mlflow_tracking_uri  # Default to ./mlruns
```

Comment:
see discussion on slack. I'm fine with making mlflow mandatory.

Reply:
This PR replaced our custom logging functions with those of MLFlow. We'd have to keep the old ones if we wanted to make it optional. I'm afraid it could become messy to manage.  

Let's try it this way and we can always come back to the older version of it if it's not working for us.